about.html

```diff
@@ -0,0 +1,145 @@
+                    <br><br>
+                    This year, the band has released several singles like “Binikini” (feat. Danny Ocean), and “Dame Un Break”.
+                    <span class="show">
+                    As a prolific touring band, Rawayana is gearing up to support their new music with touring and festival 
+                    stops in Latin America, North America (over 30 cities in Q4), and Europe throughout 2023 and 2024.
+                    </span>
+                </p>
+            </div>
+            <div class="container" id="contact">
+                <p> 
+                    Please contact us via email for any further questions about the event!
+                    <br>
+                    <a href="mailto:sadicoficial@gmail.com">rawayanaoficial@gmail.com</a>
+                </p>
+            </div>
+        </section>
+
+        <section id="summit">
+            <div class="container">
+                <div>
+                    <h1> Rawayana's tag <br>
+                    and 2023 tour Logo </h1>
+                </div>
+                <hr>
+                <div>
+                    <p>
+                    The logo of Rwayana's LA Tour 2023 was decided through fan voting from June 23rd to June 30th.
+                    </p>
+                </div>
+                <div>
+                    <img src="./Assets/event.png">
+                </div>
+                <div>
+                    <h3> See the past tours</h3>
+                </div>
+                <hr>
+                <div>
+                    <p>
+                    Take a look at the past two toors which took place in La Vega, Caracas!
+                    </p>
+                </div>
+            </div>
+        </section>
+
+        <section id="past">
+            <div class="container" id="concierto">
+                <h2>2020</h2>
+                <h5>Concierto en La Vega</h5>
+            </div>
+            <div class="container" id="concierto">
+                <h2>2020</h2>
+                <h5>Concierto en La Vega</h5>
+            </div>
+        </section>
+        
+        <section class="container" id="partner">
+            <h3>Partner</h3>
+            <hr>
+            <div class="container">
+                <img src="./Assets/partners/mozilla.png"/>
+                <img src="./Assets/partners/google.png"/>
+                <img src="./Assets/partners/naver.png"/>
+                <img src="./Assets/partners/daumkakao.png"/>
+                <img src="./Assets/partners/airbnb.png"/>
+            </div>
+        </section>
+
+        <footer class="contianer" id="cc">
+            <div id="logo">
+                <img src="/Assets/logo.png">
+            </div>
+            <div id="rights">
+                <p>
+                    2023 Creative Commons. <br>
+                    Some rights reserved.
+                </p>
+            </div>
+        </footer>
```

Comment:
- [ ] This footer on desktop screen size have a dark grey color as in the given design. kindly consider fixing it.
- [ ] Also the partner section doesn't exist on about page desktop screen so consider removing it.
- [ ] Also consider increasing the size of this footer from top and bottom as you footer size is smaller than the footer on behance design.
your design
<img width="987" alt="Screenshot 2023-06-30 at 7 02 04 PM" src="https://github.com/Arch-Noize/capstone-portfolio/assets/96683843/b460143f-a1bd-4476-9b05-1130d0026760">

given design

<img width="907" alt="Screenshot 2023-06-30 at 6 55 26 PM" src="https://github.com/Arch-Noize/capstone-portfolio/assets/96683843/a77b7ba4-d30c-4479-8b14-d60cfeeffd62">